.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+- Fixes #issue-number
+
+**Description**
+
+#### Checklist
+
+- [ ] I have made corresponding changes to the documentation.
+- [ ] I have verified the functionality in some of the neeto web-apps.
```

Comment:
Since this is an opensource project, outsiders can raise PRs.
But they can't test the functionality in neeto-*-web apps.